src/tests/EdFi.DataManagementService.Tests.E2E/Features/NaturalForeignKeyValidation.feature.disabled

```diff
@@ -0,0 +1,119 @@
+Feature: Validation of Natural and Foreign Keys
+
+        Background
+            Given there is a School
```

Comment:
Doesn't this have to be `Given the system has these "schools"` ? Same with `Session` and `Course`

Reply:
Changed